_authors/milarepa.md

```diff
@@ -0,0 +1,5 @@
+---
+author: "Milarepa"
+---
+
+Milarepa was a Tibetan master, yogi, and poet who led an inspiring life of spiritual progress and human accomplishment.
```

Comment:
In the description, give the full name (years) and tell the reader why they should care.

For example, here is Wikipedia's introduction:

> Jetsun Milarepa (Tibetan: རྗེ་བཙུན་མི་ལ་རས་པ་, Wylie: rje btsun mi la ras pa, 1028/40–1111/23) was a Tibetan siddha, who was famously known as a murderer when he was a young man, before turning to Buddhism and becoming a highly accomplished Buddhist disciple. He is generally considered one of Tibet's most famous yogis and spiritual poets

I'd also include something about his famous training under Master Marpa, and about his legendary strength and devotion... anything else you think it's essential people know about him as a figure.